### DISCUSSION TITLE
Feature request: Allow the aria-label of SearchBox input to be translated

### DISCUSSION THREAD

**Discussion by huyenltnguyen** [algolia/instantsearch#6450] [Ideas]
## Description

The `aria-label` of SearchBox's `input` element is currently:

https://github.com/algolia/instantsearch/blob/d68f02c6db8d6eed4be4330dacf246a3856bb209/packages/react-instantsearch/src/ui/SearchBox.tsx#L197

And the `translations` API is only accepting button titles:

https://github.com/algolia/instantsearch/blob/d68f02c6db8d6eed4be4330dacf246a3856bb209/packages/react-instantsearch/src/ui/SearchBox.tsx#L47-L56

I'm wondering if aria-label text could be added to the translation list, the property could probably be named `inputAriaLabel`.

There seems to be some discussions regarding this feature in the past:
- https://github.com/algolia/instantsearch/issues/5262#issuecomment-1363061239
- https://github.com/algolia/instantsearch-specs/pull/104#issuecomment-1346609493

**Comment by Haroenv** [comment]
Yes that makes sense as a first step. Happy to accept a PR that adds the key for search box (and probably also refinement list)
